src/components/Modal.jsx

```diff
@@ -23,3 +25,12 @@ Modal.propTypes = {
 Modal.defaultProps = {
   isOpen: true,
 };
+
+const ModalContents = onClickOutside(
```

Comment:
This seems fine! A simpler approach might be to eschew `onClickOutside` altogether in this case, since there’s  only one element, the `div.modal`, that we’re listening for clicks on. So you could just add an `onClick` to the outermost div, and then a second one to `.modal__contents` to stop propagation.

Reply:
Ah that's much simpler!